<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="1400" started-at="2019-03-10T11:30:53Z" finished-at="2019-03-10T11:30:55Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="1400" started-at="2019-03-10T11:30:53Z" finished-at="2019-03-10T11:30:55Z">
      <class name="com.configTest.ConfigTest1">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.configTest.ConfigTest1@6b57696f]" name="setUp" is-config="true" duration-ms="15" started-at="2019-03-10T11:30:53Z" finished-at="2019-03-10T11:30:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.configTest.DBTest@5bb21b69]" name="setUp" is-config="true" duration-ms="1" started-at="2019-03-10T11:30:53Z" finished-at="2019-03-10T11:30:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="test()[pri:0, instance:com.configTest.ConfigTest1@6b57696f]" name="test" duration-ms="21" started-at="2019-03-10T11:30:53Z" finished-at="2019-03-10T11:30:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.configTest.ConfigTest1 -->
      <class name="com.configTest.DBTest">
        <test-method status="PASS" signature="dbTest()[pri:0, instance:com.configTest.DBTest@5bb21b69]" name="dbTest" duration-ms="1313" started-at="2019-03-10T11:30:53Z" finished-at="2019-03-10T11:30:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dbTest -->
      </class> <!-- com.configTest.DBTest -->
      <class name="com.configTest.CreateJira">
        <test-method status="FAIL" signature="test1()[pri:0, instance:com.configTest.CreateJira@6b9651f3]" name="test1" duration-ms="12" started-at="2019-03-10T11:30:55Z" finished-at="2019-03-10T11:30:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.configTest.CreateJira.test1(CreateJira.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- com.configTest.CreateJira -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
